src/types/types.ts

```diff
@@ -10,6 +10,10 @@ export type Config = {
   headers?: object
 }
 
+export type Result<T> = {
```

Comment:
Just for information, what is the advantage of creating a specific object to return the keys instead of a promise?

Reply:
not sure I understood your question.
I created this type because a lot of API routes will not return their data in the `results` object. 
Same for `tasks` but I need to change it in another PR.

Reply:
Thanks!